ic-task-scheduler/src/scheduler.rs

```diff
@@ -141,16 +363,20 @@ impl<T: 'static + Task, P: 'static + UnboundedMapStructure<u32, ScheduledTask<T>
 }
 
 #[cfg(test)]
+#[allow(clippy::type_complexity)]
```

Comment:
???

Reply:
warning: very complex type used. Consider factoring parts into `type` definitions
   --> ic-task-scheduler/src/scheduler.rs:568:27
    |
568 |           fn scheduler() -> Scheduler<
    |  ___________________________^
569 | |             SimpleTaskSteps,
570 | |             StableUnboundedMap<
571 | |                 u32,
...   |
574 | |             >,
575 | |         > {
    | |_________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#type_complexity
    = note: `#[warn(clippy::type_complexity)]` on by default